README.md

```diff
@@ -97,6 +97,36 @@ Message.unread_by(current_user)
 Message.read_by(current_user)
 # => [ message1, message2 ]
 
+## Get users that have not read a given message
+user1 = User.create!
+user2 = User.create!
+
+User.have_not_read(message1)
+# => [ user1, user2 ]
+
+message1.mark_as_read! :for => user1
+User.have_not_read(message1)
+# => [ user2 ]
+
+## Get users that have read a given message
+User.have_read(message1)
+# => [ ]
```

Comment:
This is confusing. After marking message1 as read for user1 (see line 107), the result of this line has to be `[message1]`. It should be pointed out that here starts a new example.